Dockerfiles/manifests/rbac/clusterrole.yaml

```diff
@@ -28,8 +28,9 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - endpoints
+  - configmaps
   verbs:
   - get
   - update
   - create
+  - list
```

Comment:
why do we need this additional `list` verb compared to the `endpoints`?

Reply:
@mfpierre thank you for bringing this up.
For the Service Mapper (and other functions like the checkResourcesAuth) we need to list the Endpoints. This verb is above in the first section of the role. We never used to need to list the configmaps.
Now that the Leader Election relies on configmaps we need to List them.
We still need to List the endpoints though, which is why the verb was added.